ubiquibot-config-default.ts

```diff
@@ -90,5 +91,8 @@
   "enable-access-control": {
     "label": false,
     "organization": true
-  }
+  },
+  "stale-bounty-time":"0d"
 }
+
+export default DefaultConfig;
```

Comment:
```suggestion
export default DefaultConfig as readonly;
```

I'm not sure which way is correct but you can do `as readonly` or `as const` I'm pretty sure which will actually make this whole object immutable.

Reply:
Got it, it will be 
```bash
export default DefaultConfig as const
```
Is "DefaultConfig" fine or it should be changed to "DEFAULT_CONFIG"?

Reply:
DefaultConfig is fine to me

Reply:
None of them are working.
On adding "as readonly" it's giving "Type expected" error and 
On adding "as const" , it's showing "const" can be added to string, number, boolean,enum, array and object literals only.
 
Is "DefaultConfig" fine or it should be changed to "DEFAULT_CONFIG"?
@0xcodercrane mentioned "DefaultConfig" is fine and @pavlovcik asked to use "DEFAULT_CONFIG"

Reply:
ok. I couldn't find @pavlovcik 's comment because it has been already resolved.

Lets go with `DEFAULT_CONFIG` and use const.

Reply:
Use chatgpt to figure out how to make objects immutable @Wholesomebruh